### ISSUE TITLE
Define "Fun"

### ISSUE THREAD

**Issue by anhegofcherek** [prophecymud/ideas#74]
Bear with me here...

A few times now, here and elsewhere, I've found myself using the term "Fun", like (paraphrasing):

* Botting isn't "Fun"
* Having to carefully keep your stats maxed while levelling, to get the best gains, isn't "Fun"
* Levelling should be "Fun"
* Aquesting should be "Fun"

etc.

Obviously the actual notion of fun is subjective, but I think there are things we can agree are:

* Frustrating to have to do, unrewarding, uninteresting, ("Un-Fun") or
* Interesting, challenging, potentially educational ("Fun")

and I think we'd generally agree that we'd like to have more Fun and less Un-Fun (specifically using the term "Un-Fun" to differentiate from "not Fun" - anti-Fun not just absence of Fun)

I wonder if this is a useful categorisation / concept, if this post in itself is enough, and/or there are other principles worth establishing, e.g.:

1. "If something is Fun for one person but Un-Fun for someone then it is Un-Fun" (or something that encodes a notion of griefing?)
2. "If something can be easily automated, it's Un-Fun" (maybe? it's probably not especially challenging or rewarding!)

I'm not hugely sure what I'm going for here, but I'm interested in thoughts please!

**Comment by prophnoran** [comment]
Yes, this is a useful categorization to draw. I think one should avoid taking the analytic-philosophy[0] approach of trying to pin it down with a definition or write propositions that we think at least agree with our intuition -- we won't get anywhere, but instead fall into Wittgenstein's games trap [1]. Instead I would suggest a general descriptive approach: let's solicit various players to say (here, eg) what is fun *for them* (an advantage of a small playerbase). For example, here is some introspection about my preferences:

- I find the satisfaction of watching a character slowly get better fun
- I find tedious/repetitive tasks *very* not fun (but oddly I do find automating them kind of fun! Indeed, botting can be quite fun to set up, once -- but the appeal wears off pretty quickly)
- I find exploring an area fun (I especially like miniquests)
- I find competition fun (trivia, scrims, etc.)
- I am risk-averse; I don't find things to be more exciting if losing a bunch of stuff is on the line
- I find the social aspect of Prophecy (chatting etc.) fun
- I don't really like RP very well

I think that is probably not a surprising set of facts given the suggestions I make (mostly to do with reducing (what I find) tedious tasks and adding things that encourage players to explore our areas (imo one of the strongest parts of proph is our areas). It goes without saying that these are not universal preferences; in particular I suspect that I am unusually intolerant of tedium, given the popularity of very grindy RPGs in the world...

[0] The old joke: why should you never date an analyst? Before they'll say "I love you" they have to spend forty years deciding what "I" means.

[1]  From *Philosophical Investigations* (1953):
> 66. Consider for example the proceedings that we call “games”. I mean board-games, card-games, ball-games, Olympic games, and so on. What is common to them all? — Don’t say: “There must be something common, or they would not be called ‘games’ “ — but look and see whether there is anything common to all. — For if you look at them you will not see something that is common to all, but similarities, relationships, and a whole series of them at that. To repeat: don’t think, but look! — Look for example at board-games, with their multifarious relationships. Now pass to card-games; here you find many correspondences with the first group, but many common features drop out, and others appear. When we pass next to ball- games, much that is common is retained, but much is lost. — Are they all ‘amusing’ [i.e. fun]? Compare chess with noughts and crosses. Or is there always winning and losing, or competition between players? Think of patience [i.e. Solitaire]. In ball games there is winning and losing; but when a child throws his ball at the wall and catches it again, this feature has disappeared. Look at the parts played by skill and luck; and at the difference between skill in chess and skill in tennis. Think now of games like ring-a-ring-a-roses; here is the element of amusement, but how many other characteristic features have disappeared! And we can go through the many, many other groups of games in the same way; can see how similarities crop up and disappear.

> And the result of this examination is: we see a complicated network of similarities overlapping and criss-crossing: sometimes overall similarities, sometimes similarities of detail.

**Comment by Ranfir** [comment]
To indirectly answer your question, I'd like to describe the *most memorable* moments that come to mind when I reminisce about Prophecy:

## In search of fun
I first got the idea of mudding back in '93-'94 from watching Nidano and other kids play the Arctic MUD on the library computers. I didn't actually start playing until somewhere around '96 or '97. What drew me to Prophecy (along with Mortiris, Nidano, and Dori), was the browser client that made it easy to play from our school's computer lab. I played as Alexanderu, the Sendarian Warrior, and only made it to level 30-something with the limited computer time that I had. I don't really remember much else from back then, apart from Dori posing as a girl, and catfishing some guy into pleveling "her" to level 91 in one weekend. (LOL!)

### Emily Scavenger (2001/2002?)
Yes -- believe it or not, this is one of the earliest memories that stands out in my mind. This is what drew me back into the game for the second time -- finding an `(Emily) (Scavenger) A Golden Egg` back when it was worth 2qps. I even remember the exact room that I found it in:
```
The Imperial Park
  A flat expanse of beautiful green grass spreads out in all directions
here.  To the west brightly colored hedges form a barrier and to the east a
small grove of tightly nestled trees can be seen.  To the south the Grand
pavillion of Tol Honeth can be seen its surface shifting as a warm breeze
strikes against it.  An ivory bench can be seen here and a small basket used
to dispose of waste sits off to the left of it.  

[Exits: north east south]
     A wire wastebasket sits here.
     An ivory park bench rests here.
```
I don't think I'll ever forget that wire wastebasket. :)

### Bob's Quest (2002?)
Back when I played as Raife the Ranger, I participated in a Bob's quest. My memory says it was held by Ctuchik, but my notes say that Javelin held the quest. (Who knows?)

Raife was easily the crappiest character participating in the quest. I don't think I was even level 91 at the time. My melee never landed on any of the demon lords, so all I had was my lightning bolt spell before the quest mobs killed me in one melee round. (Yes, lightning **bolt**, not lightning breath.) Somehow, 5 hours into the quest, I managed to get the killing blow on Bob with a whopping 20 dmg. I was over my item limit, so someone else looted Bob's pendant, but the other players cried foul, and made sure that I got the prize for getting in the last hit. :)

### PK (2006?)
Ah!!! This was so insanely fun and addictive that I had to ask Kerowa to change all of our passwords so that I would actually do my homework! After years of scrims, I finally decided to take the plunge into PK. I still get a bit of an adrenaline surge when re-reading my old PK logs.

Of the 60-some kills I made, the one that sticks out the most was when I looted Naraj. It was definitely not a *fun* fight. Naraj had perm_sanc, an average 45 dagger, and 190+ hitdam. The fight was very one-sided until I finally managed to trap and kill him. And I definitely felt awful about looting him as well. Not fun! (Ironically, most everything I know about PK as a priest, I learned from watching Naraj's priest. Naraj is easily the best PKer I've ever fought.)

## So what is fun?
Searching for scavenger objects room by room in the mud is decidedly not "fun" for me. But finding that first Emily Golden Egg was exhilarating!

Dying for 5 hours straight in Bob's quest as a lowbie char -- ehh, not that exciting. But the promise of cool quest prizes -- definitely made the experience fun. And getting in the last blow on Bob... was totally awesome!

Leveling a character to 91 and obtaining full aqeq for the char -- not fun for me. PKing as that character once it's all been done -- **FUN**. (**Idea**: give pk_set chars a per-kill experience bonus.)

That all being said, I would argue that the tedious and un-fun parts of the experience don't necessarily detract from the overall experience. Rather, the delayed gratification from doing un-fun things can potentially make the overall experience more fulfilling for many players.
